plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -213,9 +214,16 @@ func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes,
 		return pod, "", nil, nil
 	}
 
-	// sort by name to make order deterministic
+	// check the validated policy first if provided, sort other policies by name to make order deterministic
 	// TODO(liggitt): add priority field to allow admins to bucket differently
 	sort.SliceStable(policies, func(i, j int) bool {
+		if policies[i].Name == validatedPSPHint {
```

Comment:
only do this if specMutationAllowed == false. otherwise, we want the order to be that defined by the policy creator

Reply:
Done.